.classpath

```diff
@@ -1,17 +1,46 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do you want to check in .classpath and .project? This is generated by eclipse when importing a maven project.